EasyPostTest/ErrorTest.cs

```diff
@@ -32,5 +41,27 @@ public void TestErrorLoad()
             Assert.AreEqual("address", sut.errors[0].field);
             Assert.AreEqual("Address not found", sut.errors[0].message);
         }
+
+        [TestMethod]
+        public void TestAddressError()
+        {
+            address.Create();
+            HttpException ex = null;
+            try {
+                address.Verify();
+            }
+            catch (HttpException hex)
```

Comment:
That would stand for HttpWebException it's just my habits, i'll standardize it based on your standards.